lib/uploadcare/rails/active_record/mount_uploadcare_file.rb

```diff
@@ -12,58 +14,63 @@ module ActiveRecord
 
       def build_uploadcare_file(attribute)
         cdn_url = attributes[attribute.to_s].to_s
-        return nil if cdn_url.empty?
+        return if cdn_url.empty?
 
         uuid = cdn_url.match(UUID_REGEX).to_s
-        ::Rails.cache.read(cdn_url) || Uploadcare::Rails::File.new(cdn_url: cdn_url, uuid: uuid.presence)
-      end
-
-      def log_uploadcare_error(exception, message)
-        logger.error message
-        logger.error ::Rails.backtrace_cleaner.clean(exception.backtrace).join("\n ").to_s
+        cache_key = File.cache_key(cdn_url)
+        ::Rails.cache.fetch(cache_key) do
+          Uploadcare::Rails::File.new(cdn_url: cdn_url, uuid: uuid.presence)
+        end
       end
 
       class_methods do
-        # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Lint/NestedMethodDefinition
-        def mount_uploadcare_file(attribute, _options = {})
-          define_method attribute.to_s do
+        def log_uploadcare_error(exception, message)
+          logger.error message
+          logger.error ::Rails.backtrace_cleaner.clean(exception.backtrace).join("\n ").to_s
+        end
+
+        def uploadcare_perform_file_deleting(file_uuid)
+          Uploadcare::FileApi.delete_file(file_uuid) if file_uuid
```

Comment:
Maybe we can not to define this and just use `Uploadcare::FileApi` (still the same reason - fewer methods in the AR)?
We can define only the attribute-related methods... 

Also, we can move logging to the gem (with the configurable logger - including no logger).

@dmitry-mukhin, do other client libraries have some logging system?